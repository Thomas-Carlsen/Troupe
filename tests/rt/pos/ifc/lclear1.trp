(* positive; there is sufficient clearance to 
   perform the receive *)

let val main = self () 
    val _ = spawn ( fn () => send (main, 42))
    val c = raisembox ({alice,bob})    
    val result =  rcv ({}, {alice}, [hn x => print x ] )
    val _ = debugpc ()
in  result
end
