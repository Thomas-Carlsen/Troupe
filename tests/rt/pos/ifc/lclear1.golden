2020-02-09T01:34:44.310Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
42
[31mPID:b7c4ba09-3ea4-475b-b777-7eabf978e128@{}%{}    [39m [31mPC:{}               [39m [31mBL:{alice}          [39m 
>>> Main thread finished with value: ()@{alice}%{alice}
